serverless-logic/serverless-logic-supporting-services/serverless-logic-job-service.adoc

```diff
@@ -8,4 +8,24 @@ toc::[]
 
 The Job service schedules and executes tasks in a cloud environment. Independent services implement these tasks, which can be initiated through any of the supported interaction modes, including HTTP calls or Knative Events delivery.
 
-//I will include more modules in smaller, incremental PRs to avoid having one large PR. Currently building the directory structure for the OSL content. 
\ No newline at end of file
+In {ServerlessLogicProductName}, the Job service is responsible for controlling the execution of the time-triggered actions. Therefore all the time-based states that you can use in a workflow, are handled by the interaction between the workflow and the Job service.
+
+For example, every time the workflow execution reaches a state with a configured timeout, a corresponding job is created in the Job service, and when the timeout is met, a http callback is executed to notify the workflow.
```

Comment:
Maybe we should add an example workflow here @wmedvede

Reply:
Thank you!